OpenRA.Mods.Common/Traits/Infantry/TakeCover.cs

```diff
@@ -58,17 +58,35 @@ public class TakeCover : Turreted, INotifyDamage, IDamageModifier, ISpeedModifie
 		[Sync]
 		int remainingDuration = 0;
 
-		bool IsProne => !IsTraitDisabled && remainingDuration != 0;
+		bool isProne = false;
+		bool Prone
```

Comment:
- As a boolean property this should be called `IsProne`.
 - I know @abcdefg30's request got us here, but this makes no sense as a setter-only property. It should be a method called `SetProneState` ("state" because it also sets the offset, not just the prone field).

Reply:
Indeed, if we do it this way it should have a getter as well. Seeing this you're probably right that we should go back to LieDown/StandUp or SetProneState.

Reply:
> LieDown/StandUp

Personally I don't like such wording in code. When one function is the opposite of other I much prefer using the same word for both e.g. LieDown and UnLieDown. That's why I went with Prone and UnProne

Reply:
That isn't valid English though and doesn't describe what is happening.